1.fee_bumping/fee_bumping.md

```diff
@@ -153,25 +153,217 @@ Bitcoin blockchain, such as exchanges or custodians:
 
 These issues will be discussed in more detail in later sections of this document.
 
-# Introduction to RBF
-
-TODO
-
-## Overview of what RBF is
-
-TODO
-
-## History of RBF
-
-TODO
+# Transaction replacement
+
+The concept of being able to 'replace' an unconfirmed transaction has been
+around since the very earliest days of Bitcoin. Early versions of the
+Bitcoin reference implementation [allowed transactions to be replaced][full_replacement] by
+constructing and broadcasting a replacement transaction with modified
+`nSequence` numbers in the transaction inputs.
+
+Satoshi understood the need for transaction fees to increase as the subsidy
```

Comment:
It's nice that Satoshi was so prescient, but this motivation section would be improved by adding some examples of how that has actually manifested for users today. I'm thinking that the December fee hike would yield examples for example in regard to commerce and exchange interactions.

Reply:
This is a slightly different point. This quote is about moving to an inflationary policy to a system supported entirely by tx fees. Perhaps I could strengthen the motivation in the `# Introduction` section to address your point here?

Reply:
Sure, that would also be a good place.